scripts/check_frontend_coverage.py

```diff
@@ -198,7 +198,6 @@
     'list-of-sets-of-html-strings-editor.directive.ts',
     'list-of-tabs-editor.directive.ts',
     'list-of-unicode-string-editor.directive.ts',
-    'loading-dots.directive.ts',
```

Comment:
Hi @marianazangrossi, Why was this change demanded? The frontend tests were failing for me saying this file is fully covered, please remove it from here.

Reply:
@bansalnitish The list is now a blacklist, so any file that not fully covered should be in the list

Reply:
You're right @srijanreddy98, also @bansalnitish can you please test this component?

Reply:
PTAL now @marianazangrossi

Reply:
Thanks, @bansalnitish